col_test.go

```diff
@@ -360,6 +361,65 @@ func TestColWidth(t *testing.T) {
 	convertRowHeightToPixels(0)
 }
 
+func TestAutoFitColWidth(t *testing.T) {
+	randStr := func(n int) string {
```

Comment:
Can we simplify this random utils by `strings.Repeat`? We don't care about real letters.